src/neo/Ledger/TransactionVerificationContext.cs

```diff
@@ -0,0 +1,37 @@
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+using Neo.SmartContract.Native;
+using System.Collections.Generic;
+using System.Numerics;
+
+namespace Neo.Ledger
+{
+    public class TransactionVerificationContext
```

Comment:
I see this as a more generic use as you like @erikzhang.
Looks good to be and perhaps it will be more efficient.
About the renames, perhaps this more generic may allow other inclusions/verifications in the future.